### ISSUE TITLE
URN rendering fails

### ISSUE THREAD

**Issue by andrew2net** [metanorma/pubid-iso#142]
```
> Pubid::Iso::Identifier.parse("ISO/TS 19115-3:2016").urn
     NoMethodError:
       undefined method `type' for "TS":String
     # /Users/andrej/.rvm/gems/ruby-3.0.0/gems/pubid-core-1.2.2/lib/pubid/core/renderer/base.rb:76:in `render'
     # /Users/andrej/.rvm/gems/ruby-3.0.0/gems/pubid-iso-0.2.1/lib/pubid/iso/identifier.rb:108:in `urn'
     ...
 ```

**Comment by ronaldtse** [comment]
@mico this is urgent, thanks.

**Comment by mico** [comment]
@andrew2net I just published `pubid-iso` 0.2.2, try it. There were many changes since 0.2.1
